apps/application-system/api/infra/application-system-api.ts

```diff
@@ -298,6 +298,8 @@ export const serviceSetup = (services: {
         '/k8s/api/ALTHINGI_OMBUDSMAN_XROAD_USERNAME',
       ALTHINGI_OMBUDSMAN_XROAD_PASSWORD:
         '/k8s/api/ALTHINGI_OMBUDSMAN_XROAD_PASSWORD',
+      RECYCLING_FUND_GQL_BASE_PATH:
+        '/k8s/skilavottord-ws/RECYCLING_FUND_GQL_BASE_PATH',
```

Comment:
Is this sensitive? Looks like a config option to me. Please remove from `.secrets`

Reply:
RECYCLING_FUND_GQL_BASE_PATH  is the path to the skilavottord-ws GraphQL server.
Car recycling application uses the skilavottord-ws  GraphQL server when the application is sent.

Reply:
This shouldn't be a secret. But this is the path to the skilavottord gql server which is our AWS cluster in another namespace.

@island-is/devops / @AndesKrrrrrrrrrrr can they use AWS service url or do the need to go through public path in ingress?

They probably need to grant some namespace in skilavottord-ws/infra to enable this communication?

Reply:
@AndesKrrrrrrrrrrr this should be fixed now. Using the `ref` utility to compute the appropriate svc url for AWS cluster. Also granting namepsace access in `skilavottord-ws`.